src/client/localplayer.h

```diff
@@ -71,6 +71,8 @@ class LocalPlayer : public Player
 	// Temporary option for old move code
 	bool physics_override_new_move = true;
 
+	const float m_zoom_fov_default = 15.0f; //assumed zoom FOV for zoom bypass
```

Comment:
would it be beneficial to make this configurable?

Reply:
Not 100% sure what this is actually - if it's the zoom value i guess yeah might make sense to make that configurable.

It might also be the value it returns to after zoom. I remember having trouble finding the correct value there. In which case it would probably be best to put the real one and remove that const.

Reply:
no it is the zoom value .. making it configurable